rotkehlchen/tests/unit/test_inquirer.py

```diff
@@ -435,11 +435,11 @@ def test_find_curve_lp_token_price(inquirer: 'Inquirer', blockchain: 'ChainsAggr
         # Curve.fi amDAI/amUSDC/amUSDT (am3CRV)
         ChainID.POLYGON_POS: ('0xE7a24EF0C5e95Ffb0f6684b813A78F2a3AD7D171', FVal('1.129260')),
         # crvUSDT-gauge
-        ChainID.ARBITRUM_ONE: ('0xB08FEf57bFcc5f7bF0EF69C0c090849d497C8F8A', FVal('2.024704')),
+        ChainID.ARBITRUM_ONE: ('0xB08FEf57bFcc5f7bF0EF69C0c090849d497C8F8A', FVal('2.025168')),
```

Comment:
why did this test changed?

Reply:
This is because earlier we did not have coingecko/cryptocompare id for one of its lp token `eip155:100/erc20:0xaBEf652195F98A91E490f047A5006B71c85f058d`, so they were getting queried from defillama. Now, they get queried from coingecko/cryptocompare, so price is slightly varied.